docs/Advanced-Topics.rst

```diff
@@ -77,3 +77,26 @@ Recommendations for gcc Users (MinGW, \*nix)
 --------------------------------------------
 
 -  Refer to `gcc Tips <./gcc-Tips.rst>`__.
+
+Support for Position Bias Treatment
+--------------------------------------------
+
+Often the relevance labels provided in Learning-to-Rank taks might be derived from implicit user feedback (e.g., clicks) and therefore might be biased due to their position on the screen when having been presented to a user. LightGBM uses an additional file to store positional data, like the following:
+
+::
+
+    3
+    4
+    1
+    ...
+
+It means the position of the document from the first data row, when presented to a user, was ``3``, second was ``4``, and so on. A position could be any (arbitrary) integer number and could encode the presentation of an item even in complex layouts (it is up to the LightGBM user to perform such encoding/enumeration).
+
+The position file corresponds with training data file line by line, and has one position per line.
+
+And if the name of training data file is ``train.txt``, the position file should be named as ``train.txt.position`` and placed in the same folder as the data file.
+In this case, LightGBM will load the position file automatically if it exists.
+
+Also, you can include position column in your data file. Please refer to the ``position_column`` `parameter <#position_column>`__ in above.
```

Comment:
```suggestion
Also, you can include position column in your data file. Please refer to the ``position_column`` `parameter <#position_column>`__.
```

You haven't added a `position_column` parameter yet.

<img width="516" alt="image" src="https://github.com/microsoft/LightGBM/assets/7608904/b829eeb4-d4a1-44a7-9559-03695d7502b8">

And its docs also would not be "above", but on the separate parameters page.